CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added some example values for clarity in README
 
+- Fix bug when reading `cockroach_private_endpoint_services.#.aws.service_name`.
```

Comment:
Can you add a tag? I think we'll want to release this asap.  Monday probably makes sense.

Reply:
done.